app/src/main/AndroidManifest.xml

```diff
@@ -12,22 +12,27 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".view.activities.SplashScreenActivity"
+        <activity
+            android:name=".view.activities.SplashScreenActivity"
             android:theme="@style/AppTheme.NoActionBar">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".view.activities.LoginActivity"
+        <activity
+            android:name=".view.activities.LoginActivity"
             android:theme="@style/AppTheme.NoActionBar" />
-        <activity android:name=".view.activities.SignUpActivity"
+        <activity
+            android:name=".view.activities.SignUpActivity"
             android:theme="@style/AppTheme.NoActionBar" />
         <activity android:name=".view.activities.MainActivity" />
         <activity android:name=".view.activities.RequestDetailActivity" />
         <activity android:name=".view.activities.MemberProfileActivity" />
         <activity android:name=".view.activities.SendRequestActivity" />
+        <activity android:name=".view.activities.SettingsActivity"
+            android:label="Settings"></activity>
```

Comment:
Instead of this, you can use this -> `supportActionBar?.title = getString(R.string.settings)` on the `override fun onCreate(savedInstanceState: Bundle?)` function